### PR TITLE
add Canon EOS 77D WB Presets

### PR DESCRIPTION
**Pull request by johnny-bit**
While working on #4360 I noticed (duh!) that there are no WB presets for EOS 77D.

This adds presets with finetuning.

### CODE REVIEW

**Comment by johnny-bit** [comment]
Hey @TurboGit  - should I squash the commit or is it good to go or just needs testing?

File: src/external/wb_presets.c [hunk 1]
```diff
@@ -1181,6 +1181,123 @@ const wb_data wb_preset[] =
...
+  { "Canon", "EOS 77D", WhiteFluorescent    , 0, { 1.4775390625, 1, 2.2353515625, 0 } },
+  { "Canon", "EOS 77D", CoolWhiteFluorescent, 1, { 1.4990234375, 1, 2.20703125, 0 } },
+  { "Canon", "EOS 77D", CoolWhiteFluorescent, 2, { 1.51953125, 1, 2.173828125, 0 } },
+  { "Canon", "EOS 77D", CoolWhiteFluorescent, 3, { 1.5419921875, 1, 2.142578125, 0 } },
+  { "Canon", "EOS 77D", CoolWhiteFluorescent, 4, { 1.5634765625, 1, 2.115234375, 0 } },
+  { "Canon", "EOS 77D", CoolWhiteFluorescent, 5, { 1.5849609375, 1, 2.0810546875, 0 } },
+  { "Canon", "EOS 77D", CoolWhiteFluorescent, 6, { 1.6103515625, 1, 2.056640625, 0 } },
+  { "Canon", "EOS 77D", CoolWhiteFluorescent, 7, { 1.6328125, 1, 2.02734375, 0 } },
+  { "Canon", "EOS 77D", CoolWhiteFluorescent, 8, { 1.6591796875, 1, 2.0, 0 } },
+  { "Canon", "EOS 77D", CoolWhiteFluorescent, 9, { 1.6865234375, 1, 1.97265625, 0 } },
```
Comment (johnny-bit) [inline src/external/wb_presets.c]: Urgh this part was dumb. Will squash it with clear `Fluorescent`.

Comment (johnny-bit) [reply src/external/wb_presets.c]: It actually caused dt to have wb menu split in very wrong way...

**Comment by TurboGit** [comment]
Thanks !

**Pull request merged**